crates/sui-framework/packages/sui-framework/Move.lock

```diff
@@ -2,6 +2,8 @@
 
 [move]
 version = 0
+manifest_digest = "20751505E200B1A5E766A503B01DF098239F2D7596998DA04016D9A34FC3E203"
```

Comment:
is this here by mistake on my machine? meaningful change?

Reply:
It's fine to leave it.

Reply:
Just FYI I have found that adding the manifest/deps digest here will break a bunch of tests in the sui-e2e tests. From what I can tell this is because the deps digest will change based on the path to the MoveStdlib since they include it via a different path. Something we should probably follow-up on, but for now reverting this will fix the failing sui-e2e tests on this PR.